Gopkg.lock

```diff
@@ -2,76 +2,57 @@
 
 
 [[projects]]
-  digest = "1:09a7f74eb6bb3c0f14d8926610c87f569c5cff68e978d30e9a3540aeb626fdf0"
```

Comment:
update your dep and try again

Reply:
I did that.  I don't know why it's still doing this :/